.gitignore

```diff
@@ -2,4 +2,5 @@
 .Rhistory
 .RData
 .Ruserdata
-README.html
\ No newline at end of file
+README.html
+data/downloaded_source_data/
```

Comment:
Worth adding a note in the README to explain this folder and pointing to the R script for it?

Reply:
Think I've covered this in the latest commit - feel free to correct / change if appropriate!